docs/src/Callout.doc.js

```diff
@@ -22,168 +47,346 @@ card(
         type: 'string',
         required: true,
         defaultValue: null,
-        description: [
-          'Text to render inside the callout to convey detailed information to the user. The message text has a fixed size.',
-        ],
-        href: '',
+        description: `
+          Main content of Callout. Content should be localized.
```

Comment:
Should we standarize it to `See [localization ](#localization) to learn more.`

Reply:
I ended up linking localized to the localization section